File: module_3/module_3_python/cours.py
"""
Classe Cours
Unité d'enseignement avec nom et professeur responsable.
"""


class Cours:

    def __init__(self, nom_cours, professeur_responsable):
        self.nom_cours = nom_cours
        self.professeur_responsable = professeur_responsable

    def __str__(self):
        return f"Cours : {self.nom_cours} (Prof : {self.professeur_responsable})"

    def __repr__(self):
        return f"Cours({self.nom_cours!r}, {self.professeur_responsable!r})"

File: module_3/Module_3_Java/src/main/java/com/ensta/gestion/modele/Cours.java
package com.ensta.gestion.modele;
/**
 * Unité d'enseignement.
 */

public class Cours {

    private String nom;
    private String professeur;

    public Cours(String nom, String professeur) {
        this.nom = nom;
        this.professeur = professeur;
    }

    public String getNom()        { return nom; }
    public String getProfesseur() { return professeur; }

    @Override
    public String toString() {
        return "Cours{nom='" + nom + "', professeur='" + professeur + "'}";
    }
}